sdk/storage/azure-storage-common/vcpkg.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "azure-storage-common-cpp",
-  "version-semver": "12.10.0-beta.1",
```

Comment:
you don't have to do this, but it is harmless.
I think, at some point, we should put "`9.99.999`" in all these vcpkg.json that are used in vcpkg manifest mode during the SDK build, to emphasize that the version here can be anything and does not need to be updated.

Reply:
Thanks for your suggestion! Will investigate for this and update in next release. I also compared to the azure-core, in the azure-core package, it's:

```
{
  "name": "azure-core-cpp",
  "version-string": "1.4.0-beta.1"
}
```

It seems both version-string and version-semver are to define the package-version. When I define the storage-common version-semver as `9.99.999`, Will it be a version change?